forwards_paper.tex

```diff
@@ -335,17 +330,97 @@ \subsection*{Recording the pedigree in forwards time}
 from $x$ to $L$ is $P_b$. We then complete the replacement event by incrementing
 $n$, ready to represent the next new node.
 
+This algorithm records only the topological information resulting from the
+forwards in time Wright-Fisher process, but it is straightforward to add
+mutational information. This can be done in two different ways.
+We can record mutations that occur during the simulation quite simply.
+For example, in Algorithm~\algref{W} we would generate mutations after
+we have recorded the edges joining the new node $n$ to its parents
+$P_a$ and $P_b$ in step \algref{W7}.
+For example, if we assume that infinite sites mutations
+occur at rate $\mu$ per generation then there are $\mbox{Poisson}\left((r - \ell)(\tau_p
+- \tau_c)\right)$ mutations on each edge $(\ell, r, p, c)$. Each of these mutations
+will occur on a distinct site $x$ drawn uniformly from $(\ell, r]$ and be over node $c$.
+It is straightforward to record this information during the simulation, but it
+is significantly simpler and more efficient to generate these mutations
+after the simulation has completed. In the case of infinite sites mutations
+there is no difference between generating these mutations during the forwards
+in time simulation and after it has completed. We are using precisely the same
+information.
 
-\begin{enumerate}
-\item Explain how to add mutational information. We can also do this afterwards on the
-output trees if they are nuetral.
-\item Explain that we can input these trees into an mpsrime tree sequence and they will
-work fine. Perhaps a diagram to show the kind of trees we get? (Multiroots, lots of unary nodes.
-Can use the msprime SVG API to draw these?) We could refer ahead to figure 2
-instead though.
-\item However, there is huge amount of redundancy in these. The amount of information stored
-grows linearly with time.
+\begin{figure}
+    \begin{center}
+        \includegraphics{wf-before}
+        \includegraphics{wf-after}
+    \end{center}
+    \caption{An example of a marginal genealogy from a Wright-Fisher simulation
+    with $N=5$. On the left we have the original tree including all
+    intermediate nodes and dead-ends. On the right we have the minimal tree
+    relating all of the currently-alive individuals (27--31).
+    \jk{This figure needs a little tidying up. We should combine the two SVGs
+        into a single image with some (a) (b) annotations. Also the node labels
```

Comment:
I can easily tweak these things in inkscape, but will wait 'til the last minute to do it.

Reply:
What would be great, thanks Peter!